src/main/java/io/codelex/arrays/practice/TicTacToe.java

```diff
@@ -1,17 +1,80 @@
 package io.codelex.arrays.practice;
-
 import java.util.Scanner;
 
 public class TicTacToe {
 
-    private static char[][] board = new char[3][3];
+    static char[][] board = new char[3][3];
+    static boolean won = true;
 
     public static void main(String[] args) {
-        Scanner keyboard = new Scanner(System.in);
-
         initBoard();
         displayBoard();
 
+        int i = 0;
+        while (i < 9) {
+            Scanner keyboard = new Scanner(System.in);
+
+            System.out.print("'O', choose your location (row): ");
+            int row = keyboard.nextInt();
+            System.out.print("'O', choose your location (column): ");
+            int column = keyboard.nextInt();
+
+            board[row][column] = 'O';
+            displayBoard();
+            checkWinner();
+            if (!won) break;
+
+            System.out.print("'X', choose your location (row): ");
+            int row1 = keyboard.nextInt();
+            System.out.print("'X', choose your location (column): ");
+            int column1 = keyboard.nextInt();
+
+            board[row1][column1] = 'X';
+            displayBoard();
+            checkWinner();
+            if (!won) break;
+
+            i++;
+        }
+    }
+
+    public static void checkWinner() {
+        if (
+               (board[0][0] == 'X' && board[1][1] == 'X' && board[2][2] == 'X')
```

Comment:
šo jau šķiet varētu saīsināt, ka salīdzina vai visi 3 lauki ir vienkārši vienādi nevis vai visi lauki ir 'X' vai 'O'

Reply:
Bet ja es salīdzinu laukus, nevis to, kas ir viņā iekšā, kā es zināšu kurš uzvar? 
> Visi trīs var būt == " ", "X", "O".
Es pamēģināju, kā ieteici, man nesanāca.